### PR TITLE
Kubernetes README

### PR DESCRIPTION
**Pull request by siddharthist**
See discussion on #1103, this is just that with squashed commits.

### CODE REVIEW

**Comment by siddharthist** [comment]
Fixed weird, erroneous commit.

**Comment by anners** [comment]
If this is just docs, can it be merged?

**Comment by siddharthist** [comment]
@anners Sure!

**Comment by zoza1982** [comment]
@siddharthist 
I will push some more updates and make new PR soon.
I missed the part when you get your K8's cluster up and running and log in to the mater kubectl command would not work as it needs to be configure to consume /etc/kubernetes/kubectl.config file.

File: roles/kubernetes/README.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
+Kubernetes
+==========
+
+.. versionadded:: 0.2
+
+**NOTES:** This is NOT kubernetes-mesos/Mantl framework, this is a standalone kubernetes cluster.
+
+**WARNING**: Do NOT install on top of existing Mantl cluster as it might brake it!
```
Comment (BrianHicks) [inline roles/kubernetes/README.rst]: break

**Comment by zoza1982** [comment]
Sorry for the typo :-(   and thanks for merging this

**Comment by siddharthist** [comment]
This still needs review from another team member.

**Pull request closed without merging**